skbeam/core/tests/test_correlation.py

```diff
@@ -202,6 +204,19 @@ def test_bad_images():
     assert_array_almost_equal(g2[:, 1], g2_n[:, 1], decimal=3)
 
 
+def test_one_bad_pixel(corr_setup):
+    num_levels, num_bufs, xdim, ydim, stack_size, img_stack, rois = corr_setup
+    img_stack = img_stack.astype(float)
+    g2_ref, lag_steps_ref = multi_tau_auto_corr(4, num_bufs, rois, img_stack)
+    img_stack[:, 250, 100] = np.nan
+    g2_test, lag_steps_test = multi_tau_auto_corr(4, num_bufs, rois, img_stack)
+
+    assert (g2_ref == g2_test).all()
```

Comment:
It would be safer to compare the arrays using `numpy.testing.assert_arrays_almost_equal`, but in this case the result is computed by the same function, so it is supposed to be identical. So I guess it may be left as is.